website/blog/2019-09-30-verdaccio-430-release.md

```diff
@@ -0,0 +1,101 @@
+---
+author: Juan Picado
+authorURL: https://twitter.com/jotadeveloper
+authorFBID: 1122901551
+title: Release 4.3.0
+---
+
+Verdaccio keeps growing thanks to their users. This release is a minor one we do every month, for further
+[information about our releases can be read here](https://github.com/verdaccio/contributing/blob/master/RELEASES.md).
+
+Furthermore, the info about the release is also available [at GitHub releases page](https://github.com/verdaccio/verdaccio/releases/tag/v4.3.0).
+
+We have some highlights to share:
+
+* At this stage, Docker 🐳 pulls [have grown to **5.7 million pulls**](https://dockeri.co/image/verdaccio/verdaccio).
+* We just reached **7.9k 🌟**, *would you help us to reach 10k?* Give us your star ⭐️!
+* **Blog** 🗒: Don't miss our new entry [**Managing multiples projects with Lerna and Yarn Workspaces**](https://verdaccio.org/blog/2019/09/07/managing-multiples-projects-with-lerna-and-yarn-workspaces) by [@sergiohgz](https://github.com/sergiohgz).
+* **[Monorepo](https://github.com/verdaccio/monorepo)**: Along the last months we have crafted our monorepo for grouping all our ecosystem, plugin, core and tooling packages. This does not mean Verdaccio will become monorepo, rather help us to growth without affect the main repository and easy updates or fast response to mistakes in any release.
```

Comment:
```suggestion
* **[Monorepo](https://github.com/verdaccio/monorepo)**: Along the last months we have crafted our monorepo for grouping all our ecosystem, plugin, core and tooling packages. This does not mean Verdaccio will become a monorepo, rather it will help us to grow without affecting the main repository and do easy updates or respond fast to mistakes in any release.
```